src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -249,14 +259,15 @@ public function click(Link $link)
      *
      * @param Form  $form   A Form instance
      * @param array $values An array of form field values
+     * @param bool  $isAjax Whether it's ajax or not
      *
      * @return Crawler
      */
-    public function submit(Form $form, array $values = array())
+    public function submit(Form $form, array $values = array(), $isAjax = false)
```

Comment:
adding arguments to a public method is a BC break, see https://symfony.com/bc

Reply:
I understood. I have changed the implementation.